### DISCUSSION TITLE
CypressError

### DISCUSSION THREAD

**Discussion by fazleyazdan** [cypress-io/cypress#28919] [Questions and Help]
i am doing automation of a demo site using cypress. the site is using cloudflare, so i have used cy.intercept() to deal with it, however it does not work, is there any other way to access the site and automate it.
this is the code.
            `describe ('Handling tables', () => {  
        
            beforeEach('loginfirst', () => {
            cy.intercept('GET', 'https://demo.opencart.com/admin/*', (req) => {
            req.headers['user-agent'] = 'Mozilla/5.0 (Windows NT 5.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/45.0.2454.85 Safari/537.36';
        }).as('loginRequest');

        cy.visit("https://demo.opencart.com/admin/")
        cy.get("#input-username").type('demo')
        cy.get("#input-password").type('demo')
        cy.get("button[type='submit']").click()

        cy.wait('@loginRequest')                 
        cy.get(".btn-close").click()              
        cy.get("#menu-customer>a").click()
        cy.get("#collapse-5>li:first-child").click()
    })

    it('check number of rows & columns', () => {

        cy.get(".table>tbody>tr").should('have.length', '10')                
        cy.get(".table>thead>tr>td").should('have.length', '7')             
    })
})`

this is the screenshot of error thrown by cypress
![Screenshot 2024-02-12 161132](https://github.com/cypress-io/cypress/assets/49106043/f22d716d-d7c6-4d4f-b42f-db40c6f60e84)

**Comment by shravani285** [comment]
Hi @fazleyazdan ,
Try this

cy.visit("https://demo.opencart.com/admin/",{ failOnStatusCode: false } )

ref :  "https://www.lambdatest.com/learning-hub/exception-handling-in-cypress"
